tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -164,9 +164,9 @@ source "${VENV_DIR}/bin/activate" || \
 
 
 # Install the pip file in virtual env
-pip install -v --force-reinstall ${WHL_PATH} \
-&& echo "Successfully installed pip package ${WHL_PATH}" \
-|| die "pip install (without --upgrade) FAILED"
+pip install -v --upgrade --no-deps --force-reinstall ${WHL_PATH} || \
+    die "pip install (without --upgrade) FAILED"
```

Comment:
Thanks for the suggestion for "--upgrade". But I think "--no-deps" needs to be removed, otherwise we'll run into issues related to protobuf being unavailable.  
Also, the substring "(without --upgrade )" needs to be removed. 
I'll fix these in my PR 1489 and test it thoroughly.